awesome_owl/static/src/utils.js

```diff
@@ -0,0 +1,12 @@
+/** @odoo-module **/
+
+import { useRef, onMounted } from "@odoo/owl";
+
+function useAutoFocus(element_ref){
```

Comment:
use camelcase for Javascript code
```suggestion
function useAutoFocus(elementRef){
```